tests/qnn/test_keras.py

```diff
@@ -47,13 +47,50 @@ def model(get_circuit, n_qubits, output_dim):
     return model
 
 
+@pytest.fixture
```

Comment:
Does this work without
`@pytest.mark.usefixtures("get_circuit_dm")`
?

(we have `@pytest.mark.usefixtures("get_circuit")` for `model()`)

Reply:
Yes, apparently it is mentioned in `pytest`'s documentation that `fixtures` cannot use other `fixtures` in this way. I suggest we remove the `@pytest.mark.usefixtures("get_circuit")` for `model()` too.

Reply:
AFAIK the fixture `get_circuit` or `get_circuit_dm` is getting used when parameterizing the `TestKerasLayerIntegration` and `TestKerasLayerIntegrationDM` classes respectively. Thus, we do not need to use `@pytest.mark.usefixture` for them.